### PR TITLE
summary and details format test script

### PR DESCRIPTION
**Pull request by AaronJhaj**

### CODE REVIEW

File: test-galasactl-ecosystem.sh [hunk 1]
```diff
@@ -178,6 +185,143 @@ function launch_test_on_ecosystem_with_portfolio {
...
+    --bootstrap ${bootstrap} "
+
+    info "Command is: $cmd"
+
+    output_file="runs-get-output.txt"
+    $cmd | tee $output_file
+
+
+    # Check that the full test name is output and formatted
+    cat $output_file | grep "test-name    :  ${GALASA_TEST_NAME_LONG}"
```
Comment (techcobweb) [inline test-galasactl-ecosystem.sh:291]: You might like to use some wild-cards in there...
the number of spaces could be variable...
eg: grep "test-name[ \t]*:[ \t]+${GALASA_TEST_NAME_LONG}" ?

File: test-galasactl-ecosystem.sh [hunk 2]
```diff
@@ -178,6 +185,143 @@ function launch_test_on_ecosystem_with_portfolio {
...
+    rc=$?
+    # We expect a return code of '0' because the ecosystem should be able to find this test as we just ran it.
+    if [[ "${rc}" != "0" ]]; then 
+        error ""
```
Comment (techcobweb) [inline test-galasactl-ecosystem.sh:295]: What error is that ?

File: test-galasactl-ecosystem.sh [hunk 3]
```diff
@@ -178,6 +185,143 @@ function launch_test_on_ecosystem_with_portfolio {
...
+        # We expect a return code of '0' because the header name should be output.
+        if [[ "${rc}" != "0" ]]; then 
+            error "Did not find header $header in details output"
+            exit 1
+        fi
+    done  
+
+    #check methods start on line 13 - implies other test details have outputted 
+    line_count=$(grep -n "method" $output_file | head -n1 | sed 's/:.*//')
+    expected_line_count="13"
```
Comment (techcobweb) [inline test-galasactl-ecosystem.sh:315]: Can the 13 be some constant defined near the testcase name ?
eg: GALASA_TEST_RUN_GET_EXPECTED_DETAILS_LINE_COUNT="13"

**Review by techcobweb — APPROVED** [review]
Approved subject to the suggested changes...

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**